Makefile

```
@@ -51,13 +51,16 @@ dev:
 	pdm install
 	if [ "$(CI)" != "true" ] && command -v pre-commit > /dev/null 2>&1; then pre-commit install; fi
 
+prerequisites:
+	pipx install --force mypy[reports]==1.10.0
+
 ########################################################################################
 # Lint and pre-commit
 ########################################################################################
 
 # Check lint with mypy.
 mypy:
```

Whether should it add its prerequisites?
```suggestion
mypy: prerequisites
```